README.md

```diff
@@ -96,6 +98,23 @@ To run the tests locally without docker:
 ./run_tests.sh local
 ```
 
+### Run
+
+You can also run `dcrd` with docker:
+
+Volumes:
+
+- `/etc/dcrd` hold `config`, `rpc.cert ` and `rpc.key` if you want to set any of these.
+- `/var/lib/dcrd` where is stored the blockchain
```

Comment:
where is stored => is where the blockchain is stored

I'd say more fully, "is where the persistent data such as the blockchain is stored"

Reply:
fixed, here and https://github.com/decred/dcrwallet/pull/999/commits/e579780c8ed2f4ecc0fe0725b040836c3abae278